### PR TITLE
Issue 5635: Added new css to download page

### LINKED ISSUE #5635: Update d.defold.com template with defold.com css
* The current formatting of d.defold.com is very hard to read and find your way.
* We also want editor links
* I also want to quickly find bob.jar (I use this page a _lot_)

### PR DESCRIPTION
**Pull request by JCash**
Added editor links
Sorted list with editor+bob at the top
Added category names
Added category icons

### CODE REVIEW

File: scripts/build.py [hunk 1]
```diff
@@ -1407,7 +1407,7 @@ def release(self):
             model['releases'] = s3.get_tagged_releases(self.get_archive_path())
             model['has_releases'] = True
         else:
-            model['releases'] = s3.get_single_release(self.get_archive_path(), self.version, self._git_sha1())
+            model['releases'] = [s3.get_single_release(self.get_archive_path(), self.version, self._git_sha1())]
```
Comment (JCash) [inline scripts/build.py:1410]: Make sure "releases" refers to an array.

File: scripts/resources/downloads.html [hunk 2]
```diff
@@ -1,85 +1,211 @@
...
-			padding: 40px 15px;
-			text-align: center;
-		}
-	</style>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Defold Downloads</title>
+    <link href='//fonts.googleapis.com/css?family=Open+Sans:400,300' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="//d.defold.com/static/bootstrap/css/bootstrap.min.css">
```
Comment (JCash) [inline scripts/resources/downloads.html:9]: I left the old one...

File: scripts/resources/downloads.html [hunk 3]
```diff
@@ -1,85 +1,211 @@
...
+    <link rel="stylesheet" href="https://defold.com/css/defold.css">
```
Comment (JCash) [inline scripts/resources/downloads.html:10]: ...and imported the newer one.

File: scripts/resources/downloads.html [hunk 4]
```diff
@@ -1,85 +1,211 @@
...
+        .navbar-inner {
+            background-color: var(--defold-blue-darker);
+            background-image: none;
+        }
+        .btn-primary {
+            background-color: var(--defold-blue-darker);
+        }
+        .btn-primary:hover {
+            background-color: var(--defold-blue);
+        }
```
Comment (JCash) [inline scripts/resources/downloads.html:29]: It mostly worked out of the box, but I had to override some colors.

File: scripts/resources/downloads.html [hunk 5]
```diff
@@ -1,85 +1,211 @@
...
+            {{/releases}}
+        </script>
 
 		<script>
-			var model = %(model)s
-			var output = Mustache.render($('#templ-releases').html(), model);
-			$("#releases").html(output);
+			var model = %(model)s;
+
+            function createGroups(releases) {
```
Comment (JCash) [inline scripts/resources/downloads.html:100]: Perhaps not the most efficient, but it gets the job done.

**Review by britzl — APPROVED** [review]
Me like!!!

**Pull request merged**